src/lib/rest/RestService.cpp

```diff
@@ -69,7 +69,7 @@ RestService*                     restBadVerbV          = NULL;
 
 /* *****************************************************************************
 *
-* restServiceGet -
+* restServiceVectorGet -
```

Comment:
This refactor is hard to follow... could you summarize the changes a little bit, pls?

Reply:
Well, this very function only changes name.
the problem comes later, in the following function.

what is changed is that i no longer lookup the rest service in the rest Service vector, as that is already done, and saved in ciP->restserviceP.

the diff is really messy ... :(